openhtf/util/conf.py

```diff
@@ -446,6 +446,22 @@ def _asdict(self):
         retval[key] = value
     return retval
 
+  @property
+  def help_text(self):
+    """Return a string with all config keys and their descriptions."""
+    result = []
+    for name in sorted(self._declarations.keys()):
+      result.append(name)
+      result.append('-' * len(name))
+      if self._declarations[name].description:
+        result.append(self._declarations[name].description.strip())
+      else:
+        result.append('(no description found)')
```

Comment:
I think it's worth adding a line for `default_value` when present

Maybe:
```
plug_teardown_timeout_s
-----------------------
Timeout (in seconds) for each plug tearDown function if > 0; otherwise, will wait an unlimited time.
Default value: 0


station_api_bind_address
------------------------
(no description found)
Default value: '0.0.0.0'
```
(With added quotes to identify string values)

Reply:
+1